keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterSpec.kt

```diff
@@ -127,7 +128,8 @@ data class ClusterSpec(
   override val maxDiffCount: Int? = 2,
   @JsonIgnore
   // Once clusters go unhappy, only retry when the diff changes, or if manually unvetoed
-  override val unhappyWaitTime: Duration? = null
+  override val unhappyWaitTime: Duration? = null,
+  override val clusterHealth: ClusterHealth? = null
```

Comment:
I think it will be pretty confusing to have both a `health` and a `clusterHealth` keys at the same level in the spec. Any chance we could merge the two together?

Reply:
That's a good point. Thinking about it further, should this be moved under the deploy strategy? It's fundamentally related to that (i.e. when do we consider the deploy successful)?